interfaces/builtin/networkmanager.go

```diff
@@ -0,0 +1,360 @@
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/login1
+    member=PrepareForSleep
+    interface=org.freedesktop.login1.Manager
+    peer=(label=unconfined),
+
+# Allow access to wpa-supplicant for managing WiFi networks
+dbus (receive, send)
+    bus=system
+    path=/fi/w1/wpa_supplicant1{,/**}
+    interface=fi.w1.wpa_supplicant1*
+    peer=(label=unconfined),
+dbus (receive, send)
+    bus=system
+    path=/fi/w1/wpa_supplicant1{,/**}
+    interface=org.freedesktop.DBus.**
+    peer=(label=unconfined),
+`)
+
+var networkManagerConnectedPlugAppArmor = []byte(`
+# Description: Allow using NetworkManager service. Reserved because this gives
+#  privileged access to the NetworkManager service.
+# Usage: reserved
+
+#include <abstractions/dbus-strict>
+
+# Allow all access to NetworkManager service
+dbus (receive, send)
+    bus=system
+    path=/org/freedesktop/NetworkManager{,/**}
+    interface=org.freedesktop.NetworkManager*
+    peer=(label=snap.network-manager.*),
+
+dbus (receive)
+    bus=system
+    path=/
+    interface=org.freedesktop.DBus.ObjectManager
+    peer=(label=snap.network-manager.*),
+
+dbus (receive, send)
+    bus=system
+    path=/org/freedesktop/NetworkManager{,/**}
+    interface=org.freedesktop.DBus.*
+    peer=(label=snap.network-manager.*),
+`)
+
+var networkManagerPermanentSlotSecComp = []byte(`
+# Description: Allow operating as the NetworkManager service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+accept
+accept4
+bind
+connect
+getpeername
+getsockname
+getsockopt
+listen
+recv
+recvfrom
+recvmmsg
+recvmsg
+send
+sendmmsg
+sendmsg
+sendto
+setsockopt
+shutdown
+socketpair
+socket
+# Needed for keyfile settings plugin to allow adding settings
+# for different users.
+# FIXME: adjust after seccomp argument filtering lands (LP: #1446748)
+chown
+chown32
+fchown
+fchown32
+fchownat
+lchown
+lchown32
+`)
+
+var networkManagerConnectedPlugSecComp = []byte(`
+# Description: Allow using NetworkManager service. Reserved because this gives
+#  privileged access to the NetworkManager service.
+# Usage: reserved
+
+# Can communicate with DBus system service
+connect
+getsockname
+recv
+recvmsg
+recvfrom
+send
+sendto
+sendmsg
+socket
+`)
+
+var networkManagerPermantedSlotDBus = []byte(`
+<policy user="root">
+		<allow own="org.freedesktop.NetworkManager"/>
+		<allow send_destination="org.freedesktop.NetworkManager"/>
+
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.PPP"/>
+
+		<allow send_interface="org.freedesktop.NetworkManager.SecretAgent"/>
+
+		<!-- Allow NM to talk to known VPN plugins; due to a bug in
+				the D-Bus daemon, when a plugin is installed and the user
+				immediately tries to use it, the VPN plugin's rules aren't
+				always loaded into dbus-daemon.  Those rules allow NM to
+				talk to the plugin.  Oops.  Work around that by explicitly
+				allowing NM to talk to VPN plugins here.
+			-->
```

Comment:
Not sure if this policy file was updated using the 1.2 release?  In the latest upstream version, the above comment actually reads:
          
<!-- These are there because some broken policies do                                                                                    
       <deny send_interface="..." /> (see dbus-daemon(8) for details).                                                                    
       This seems to override that for the known VPN plugins.                                                                             
   -->